doc/devel/documenting_mpl.rst

```diff
@@ -411,16 +243,17 @@ google docs to the mplgithub account.
 Referring to Matplotlib documents
 =================================
 
-In the documentation, you may want to include to a document in the Matplotlib
-sources, e.g., a license file or an image file from :file:`mpl-data`, refer to
-it via a relative path from the document where the rst file resides, e.g., in
-:file:`users/navigation_toolbar.rst`, we refer to the image icons with::
+In the documentation, you may want to include to a document in the
+Matplotlib src, e.g., a license file or an image file from `mpl-data`,
+refer to it via a relative path from the document where the rst file
+resides, e.g., in :file:`users/navigation_toolbar.rst`, you can refer to the
+image icons with::
 
     .. image:: ../../lib/matplotlib/mpl-data/images/subplots.png
 
-In the :file:`users` subdirectory, if I want to refer to a file in the
-mpl-data directory, I use the symlink directory.  For example, from
-:file:`customizing.rst`::
+In the `users` subdirectory, if you want to refer to a file in the mpl-data
+directory, you can use the symlink directory.  For example, from
```

Comment:
I don't think there's a symlink anymore?

Reply:
https://github.com/matplotlib/matplotlib/blob/7ddc73afeca3653bb706225c4d62492583bd88c7/lib/mpl_examples#L1

In master it looks like there still is